roles/vdm/defaults/main.yaml

```diff
@@ -45,17 +46,17 @@ V4_CFG_TLS_GENERATOR: openssl # [cert-manager,openssl]
 V4_CFG_CONSUL_ENABLE_LOADBALANCER: false
 V4_CFG_EMBEDDED_LDAP_ENABLE: false
 V4_CFG_SSSD: null
-V4_CFG_SITEDEFAULT: "{{'files/sitedefault.yaml' if not V4MT_ENABLE else 'files/sitedefault-mt.yaml'}}"
+V4_CFG_SITEDEFAULT: "{{ 'files/sitedefault.yaml' if not V4MT_ENABLE else 'files/sitedefault-mt.yaml' }}"
 
-V4_CFG_CAS_RAM: null
-V4_CFG_CAS_CORES: null
+V4_CFG_CAS_RAM:
+V4_CFG_CAS_CORES:
```

Comment:
Not sure why these are now set to the empty value and not null explicitly. Is this a linter item? And will these new values always default to null if not assigned?

Reply:
@thpang, they should still be set to null, this looks like a case that ansible-lint updated the file content when I re-ran it to verify my updates, I'll restore the origin null values as this was  an unintended change.